src/ActionsMenu.scss

```diff
@@ -0,0 +1,33 @@
+.ActionsMenu__header {
+    position:relative;
+    background-color: #cecccc;
+    margin-right: 0;
+    padding-bottom: 0.05rem;
+    padding-top: 0.05rem;
+    text-align:center;
+}
+
+.ActionsMenu__header-heading {
+    font-size: inherit;
+    font-weight: bold;
+    padding-top: 0.1rem;
+}
+
+.ActionsMenu__menu {
+    background-color: #ffffff;
+    border-color: #cecccc;
+    border-style: none solid solid solid;
+    left: 8rem;
+    overflow: visible;
+    position: absolute;
+    top: 1.5rem;
```

Comment:
In the design, it seems like the menu starts at the same height as the Action header, excluding the border width, so I would say we can give negative border width for top. 
`
top: -0.3rem;
`